packaging/func-e.wxs

```diff
@@ -0,0 +1,90 @@
+             https://docs.microsoft.com/en-us/windows/win32/msi/released-versions-of-windows-installer
+
+             InstallScope="perUser" is default (Id="ALLUSERS" doesn't work on msitools 0.101) -->
+        <Package Id="*" Keywords="Installer" Description="func-e installer" Manufacturer="Tetrate"
+                 InstallerVersion="310" Languages="1033" Compressed="yes" SummaryCodepage="1252"/>
+
+        <!-- func-e.exe is embedded inside the installer: func-e.msi
+             Cabinet="FuncE.cab" because light.exe fails on "func-e.cab" -->
+        <Media Id="1" Cabinet="FuncE.cab" EmbedCab="yes"/>
+
+        <Upgrade Id="$(var.UpgradeCode)">
+            <UpgradeVersion Minimum="$(var.Version)" OnlyDetect="yes" Property="NEWERVERSIONDETECTED"/>
+            <UpgradeVersion Minimum="0.6.0" Maximum="$(var.Version)" IncludeMinimum="yes" IncludeMaximum="no"
+                            Property="OLDERVERSIONBEINGUPGRADED"/>
+        </Upgrade>
+        <Condition Message="A newer version of func-e is already installed.">NOT NEWERVERSIONDETECTED</Condition>
+
+        <!-- Use standard properties as it is easier to comment values vs XML attributes.
+             See https://docs.microsoft.com/en-us/windows/win32/msi/property-reference -->
+        <Property Id="ARPURLINFOABOUT" Value="https://func-e.io"/>
+
+        <!-- There's currently no use case for control-panel based repair or modify -->
+        <Property Id="ARPNOREPAIR" Value="1"/>
+        <Property Id="ARPNOMODIFY" Value="1"/>
+
+        <!-- On Windows, when Id="TARGETDIR" Name="SourceDir": it will error otherwise -->
+        <Directory Id="TARGETDIR" Name="SourceDir">
+            <!-- ProgramFiles64Folder because Envoy is always 64-bit builds -->
+            <Directory Id="ProgramFiles64Folder">
+                <Directory Id="INSTALLDIR" Name="func-e">
+                    <Component Id="FuncEBin" Guid="*">
+                        <RemoveFolder Id="INSTALLDIR" On="uninstall"/>
+                        <!-- KeyPath="yes" - means this will be looked at to see if it is already installed -->
+                        <File Id="ExecutableFile" Name="func-e.exe" KeyPath="yes" Source="$(var.Bin)"/>
+                    </Component>
+                </Directory>
+            </Directory>
+        </Directory>
+
+        <Feature Id="Complete" Level="1">
+            <ComponentRef Id="FuncEBin"/>
+        </Feature>
+
+        <!-- Add func-e to the default PATH once installation succeeds.
+             This is an alternative to <Component><Environment> which doesn't work on wixl 0.101 (OS/X) -->
+        <Property Id="setx" Value="setx.exe"/>
+        <CustomAction Id="ChangePath" ExeCommand="PATH &quot;%PATH%;[INSTALLDIR]&quot;" Property="setx"
+                      Execute="deferred"/>
+        <InstallExecuteSequence>
+            <RemoveExistingProducts Before="InstallInitialize"/>
+            <!-- TODO: this doesn't cleanup the path addition on uninstall -->
```

Comment:
I will likely not address this TODO as I spent an hour and in worst case we leave func-e in the path post-install. I verified it doesn't redundantly add it on upgrade. This is the same impl as nomad, so if they manage to improve, so could we.

Reply:
resolved with a comment about an about to land feature that solves this